pmm-app/tests/pages/remoteInstancesPage.js

```diff
@@ -4,8 +4,8 @@ module.exports = {
 
     accessKey: process.env.AWS_ACCESS_KEY_ID,
     secretKey: process.env.AWS_SECRET_ACCESS_KEY,
-    usernameRDSMySQL: process.env.pmm-dev-mysql-remote-user,
-    passwordRDSMySQL: process.env.pmm-dev-remote-password,
+    usernameRDSMySQL: process.env.REMOTE_AWS_MYSQL_USER,
```

Comment:
AWS_MYSQL_WHATERVER can only be remote )
I'd suggest not to infer it can be local, and rename it to AWS_MYSQL_USER and AWS_MYSQL_PASSWORD.